src/tests/suite/daos_base_tx.c

```diff
@@ -74,7 +72,6 @@ dtx_check_replicas(const char *dkey, const char *akey, const char *msg,
 		print_message("%s: rep %d, result %d, size %lu/%lu\n",
 			      msg, i, req->result, size, req->iod[0].iod_size);
 
-		assert_int_equal(req->result, 0);
```

Comment:
why have you removed this and others in this file?

Reply:
They are redundant because:

1) The "req->result" has been checked inside lookup_single().
2) We will compare the data after the fetch.